content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,355 @@
+SPIRE can be configured as a source of cryptographic identities for Istio workloads through an integration with [Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret).
+Istio can detect the existence of a UNIX domain socket that implements the Envoy SDS API on a defined socket path, allowing Envoy to communicate and fetch identities directly from it.
+
+This integration with SPIRE provides flexible attestation options not available with the default Istio identity management while harnessing Istio's powerful service management.
+For example, SPIRE's plugin architecture enables diverse workload attestation options beyond the Kubernetes namespace and service account attestation offered by Istio.
+SPIRE's node attestation extends attestation to the physical or virtual hardware on which workloads run.
+
+## Install SPIRE
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SPIRE up and running:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml
+{{< /text >}}
+
+This will deploy SPIRE into your cluster, along with two additional components: the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) — used to share the SPIRE Agent's UNIX domain socket with the other
+pods throughout the node — and the [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar), a facilitator that performs automatic workload registration
+within Kubernetes. See [Install Istio](#install-istio) to configure Istio and integrate with the SPIFFE CSI Driver.
+
+### Option 2: Configure a custom SPIRE installation
+
+See the [SPIRE's Quickstart for Kubernetes guide](https://spiffe.io/docs/latest/try/getting-started-k8s/) 
+to get started deploying SPIRE into your Kubernetes environment. See [SPIRE CA Integration Prerequisites](#spire-ca-integration-prerequisites)
+for more information on configuring SPIRE to integrate with Istio deployments.
+
+#### SPIRE CA Integration Prerequisites
+
+These configuration requirements are necessary to successfully integrate Istio with SPIRE:
+
+1. Access [SPIRE Agent reference](https://spiffe.io/docs/latest/deploying/spire_agent/#agent-configuration-file) and
+   configure the SPIRE Agent socket path to match the Envoy SDS defined socket path.
+
+{{< text plain >}}
+socket_path = "/run/secrets/workload-spiffe-uds/socket"
+{{< /text >}}
+
+2. Share the SPIRE Agent socket with the pods within the node by deploying the
+   [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
+
+3. See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+
+## Install Istio
+

```

Comment:
Something like that?

```
{{< warning >}}
Note that you must deploy SPIRE before installing Istio into your environment so that Istio can detect it as a CA. 
{{< /warning >}}
```